doc/source/wsrep-provider-index.rst

```diff
@@ -451,6 +451,31 @@ the :file:`galera.cache` file which is used as source for |IST|. The bigger the
 value of this variable, the better are chances that the re-joining node will
 get IST instead of |SST|.
 
+.. variable:: gcomm.thread_prio
+
+Using this option, you can raise the priority of the gcomm thread to a higher
+level than it normally uses. 
+
```

Comment:
Looks good to me. But, upstream documentation has the below line which tells about when to use this variable. I feel we should also let the users know about it. What do you say?
```
You may find this useful in situations where Galera Cluster threads do not receive sufficient CPU time, due to competition with other MySQL threads. In these cases, when the thread scheduler for the operating system does not run the Galera threads frequently enough, timeouts may occur, causing the node to drop from the cluster.
```

Reply:
Updated.